### ISSUE TITLE
emacs key-bindings not work in vi's insert mode after config

### ISSUE THREAD

**Issue by owarai** [fish-shell/fish-shell#7464]
use vi-mode key-bindings, also enable emacs key-bindings in vi's insert mode seems not work, like ctrl+k etc. 

also checked this PR: [docs correct](https://github.com/fish-shell/fish-shell/pull/7274), but after adjust like that, vi key-bindings will not work.

### env info

fish version: 3.1.2

system: arch linux

terminal: alacritty

### How to reproduce:

my minimal config:

config.fish

```
set fish_greeting
```

functions/fish_user_key_bindings.fish

```
function fish_user_key_bindings
    fish_default_key_bindings -M insert
    fish_vi_key_bindings insert
end
```

**Comment by faho** [comment]
Check that PR again, it changes the order.

You need to do vi-bindings *first*, and *then* add on the emacs bindings.

That's exactly what that PR corrected.

**Comment by owarai** [comment]
after adjust like that, your vi-mode key works? like <Esc>, I will stcuk in insert mode.

**Comment by faho** [comment]
That works for me.

**Comment by faho** [comment]
Oh, no, okay, now I see what you mean.

The instructions in that PR weren't working either, so we fixed it in 021d4157ca5e5fcc6f505adffbfe392fe754e9d6.

The actual instructions for getting hybrid bindings:

```fish
function fish_user_key_bindings
    fish_default_key_bindings -M insert
    fish_vi_key_bindings --no-erase insert
end
```

**Comment by owarai** [comment]
this works! Thank you very much.
